internal/test/framework/windowsvm.go

```diff
@@ -104,6 +107,43 @@ func newWindowsVM(imageID, instanceType string, credentials *types.Credentials,
 	return w, nil
 }
 
+func (w *testWindowsVM) CopyFileTo(filePath, remoteDir string) error {
```

Comment:
Actually the implementation looks to be the same as `WindowsVM` interface's `CopyFile`. Why do we have 2 separate implementations of the same func?